fe/src/components/Navbar/Navbar.tsx

```diff
@@ -41,39 +41,39 @@ const Navbar = () => {
         color="blue-gray"
         className="p-1 font-normal"
       >
-        <a href="#" className="flex items-center">
-          Pages
-        </a>
+        <Link href="#" className="flex items-center">
+          Car Services
+        </Link>
       </Typography>
       <Typography
         as="li"
         variant="small"
         color="blue-gray"
         className="p-1 font-normal"
       >
-        <a href="#" className="flex items-center">
-          Account
-        </a>
+        <Link href="#" className="flex items-center">
+          About Us
```

Comment:
It is more convenient to have sections like `Privacy policy` or even `About us` in footer.

Reply:
I see it as a separate task, because navbar has to be dynamic based on user role